src/core/getExports.js

```diff
@@ -236,18 +236,19 @@ export default class ExportMap {
     if (this.reexports.has(name)) return true
 
     // default exports must be explicitly re-exported (#328)
+    let foundInnerMapName = false
     if (name !== 'default') {
-      for (let dep of this.dependencies.values()) {
-        let innerMap = dep()
+      this.dependencies.forEach((dep) => {
```

Comment:
I learned while doing this that `Maps` and `Sets` are very fundamentally not `Arrays`. So, no `some` and not even a clear way to utilize the basic `for` statement and break out of it (i.e. you can't access them in an indexed fashion).

I suspect the overhead of converting to an array just to use some would be greater than the gain.